### ISSUE TITLE
Time scale axes

### ISSUE THREAD

**Issue by xakod** [Profactor/cv-plot#33]
Does cv-plot support time scale axes? I cannot find smth related. I found only custom drawables. Does it enough to reimplement xaxis as time series axis and vertical grid as time vertical grid for supporting time scale axes? Or is there an easier way? 
Thanx

**Comment by wpalfi** [comment]
No, its not included. And you are right: You should replace `xAxis` (and `VerticalGrid`) with your custom drawables and create them as  in `makePlotAxes()`.
